app/src/features/rules/components/RulesList/components/RulesTable/hooks/useRuleTableColumns.tsx

```diff
@@ -145,6 +154,7 @@ const useRuleTableColumns = (options: Record<string, boolean>) => {
             <Switch
               size="small"
               checked={isRuleActive}
+              disabled={checkIsRuleGroupDisabled(rule)}
```

Comment:
the variable is wrongly named `rule` here, should be called something like `record`